.azure_pipelines/docker_images/scripts/install_mkl.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+set -ex
+
+# MKL
+MKL_VERSION=2020.0
+MKL_BUILD=166
+mkdir -p /opt/intel/lib
+pushd /tmp
+curl -fsSL https://anaconda.org/intel/mkl-static/${MKL_VERSION}/download/linux-64/mkl-static-${MKL_VERSION}-intel_${MKL_BUILD}.tar.bz2 | tar xjv
+mv lib/* /opt/intel/lib/
+curl -fsSL https://anaconda.org/intel/mkl-include/${MKL_VERSION}/download/linux-64/mkl-include-${MKL_VERSION}-intel_${MKL_BUILD}.tar.bz2 | tar xjv
+mv include /opt/intel/
```

Comment:
This looks like copy-n-paste from https://github.com/pytorch/builder/blob/master/common/install_mkl.sh
Two suggestions in that regards:
 - Can we just use the script from builder repo
 - If not, can we put it in some sort of a common folder (and indicate in the comment that it is a copy of a script from builder repo)

Reply:
That is true. We can use the script from the builder repo or place it in a common folder. To use it from the builder repo, I can do `wget https://raw.githubusercontent.com/pytorch/builder/master/common/install_mkl.sh`.